src/main/java/in/kiruba/model/User.java

```diff
@@ -0,0 +1,81 @@
+package in.kiruba.model;
+
+public class User {
+	private String name;
+	private String password;
+	private String email;
+	private int age;
+	private String mobileNo;
```

Comment:
Use appropriate datatypes for mobileNo and aadharcard - ( use Long)

Reply:
Added appropriate variable names.